apps/projects/app/components/Card/Issue.js

```diff
@@ -71,68 +72,95 @@ const Issue = ({
+      <div style={{ padding: '10px' }}>
+        <Checkbox checked={isSelected} onChange={onSelect} />
+      </div>
+
+      <IssueData>
+        <ClickArea onClick={onClick} />
+
+        <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+          <Text color={theme.textSecondary} size="small">
             {repo} #{number}
           </Text>
+          {workStatus !== 'fulfilled' && (
+            <ContextMenu>
+              <BountyContextMenu
+                work={work}
+                workStatus={workStatus}
+                requestsData={requestsData}
+                onAllocateSingleBounty={onAllocateSingleBounty}
+                onSubmitWork={onSubmitWork}
+                onRequestAssignment={onRequestAssignment}
+                onReviewApplication={onReviewApplication}
+                onReviewWork={onReviewWork}
+              />
+            </ContextMenu>
+          )}
         </div>
-        {(BOUNTY_STATUS[workStatus]) && (
-          <IssueDetails>
-            <Text size="small" color={theme.textTertiary}>
-              <span style={{ marginRight: '15px' }}>
-                {expLevel}
-                {dot}
-                {balance > 0 ? BOUNTY_STATUS[workStatus] : BOUNTY_STATUS['fulfilled']}
-                {dot}
-                  Due {DeadlineDistance(deadline)}
-              </span>
-              {labels.totalCount ? labelsBadges(labels) : ''}
-            </Text>
-          </IssueDetails>
-        )}
-      </IssueDesc>
-      <BalanceAndContext>
-        {balance > 0 && (
-          <Badge
-            style={{ padding: '10px', marginRight: '20px', textSize: 'large' }}
-            background={BOUNTY_BADGE_COLOR[workStatus].bg}
-            foreground={BOUNTY_BADGE_COLOR[workStatus].fg}
-          >
-            {balance + ' ' + symbol}
-          </Badge>
-        )}
-        {workStatus !== 'fulfilled' && (
-          <ContextMenu>
-            <BountyContextMenu
-              work={work}
-              workStatus={workStatus}
-              requestsData={requestsData}
-              onAllocateSingleBounty={onAllocateSingleBounty}
-              onSubmitWork={onSubmitWork}
-              onRequestAssignment={onRequestAssignment}
-              onReviewApplication={onReviewApplication}
-              onReviewWork={onReviewWork}
-            />
-          </ContextMenu>
+
+        <IssueTitleDetailsBalance>
+          <IssueTitleDetails>
+            <IssueTitle>
+              {title}
+            </IssueTitle>
+
+            {(balance > 0) && (
+              <Text.Block size="small" color={theme.textTertiary}>
+                <span style={{ marginRight: '15px' }}>
+                  {BOUNTY_STATUS[workStatus]}
+                  {dot}
+                  {expLevel}
+                  {dot}
+                  Due in {DeadlineDistance(deadline)}
+                </span>
+              </Text.Block>
+            )}
+            {(balance > 0) && (
```

Comment:
@stellarmagnet we reviewed this "slots" and "allocations" part for the initial release right? If so @rkzel this second text block should be removed

Reply:
I've commented it out for now.